### ISSUE TITLE
In verbb\navigation\services\navs::handleChangedNav() possible edge case results in error

### ISSUE THREAD

**Issue by lthurston** [verbb/navigation#266]
**Description**

We've encountered a case where $oldRecord->siteSettings is not set (see api/vendor/verbb/navigation/src/services/Navs.php line 317), resulting in a foreach loop over a null value on line 321:

``` 
       // When enabling/disabling sites
        if (Craft::$app->getIsMultiSite()) {
            // Has the sites been changed?
            $oldSiteSettings = Json::decode($oldRecord->siteSettings);
            $newSiteSettings = Json::decode($navRecord->siteSettings);

            // Removed sites
            foreach ($oldSiteSettings as $key => $value) {
                if (!isset($newSiteSettings[$key])) {
                    // Nothing for now
                }
            }
```

$oldSiteSettings should be checked to verify it's an array, or typecast to enforce that it's an array.

**Steps to reproduce**

This is difficult to provide clear information on, but don't think steps to reproduce are necessary to see that it _could_ happen and therefore guardrails should be in place to prevent it.

**Additional info**
- Plugin version: 1.4.18
- Craft version: 3.6.17
- Multi-site: Yes
- Propagate nodes: Not sure

I'll provide a PR.

**Comment by lthurston** [comment]
Well, I'll be damned, this is fixed already. Not sure why we don't have this version of the plugin, but that's a me problem. Disregard!

**Comment by engram-design** [comment]
Yep, sorry about that, just waiting on some other support requests!

**Comment by engram-design** [comment]
Fixed in [1.4.19](https://github.com/verbb/navigation/releases/tag/1.4.19)
